core/templates/pages/exploration-editor-page/services/router.service.spec.ts

```diff
@@ -420,7 +411,14 @@ describe('Router Service', function() {
   it('should save pending changes', function() {
     var broadcastSpy = spyOn($rootScope, '$broadcast').and.callThrough();
     RouterService.savePendingChanges();
-    $timeout.flush();
     expect(broadcastSpy).toHaveBeenCalledWith('externalSave');
   });
+
+  it('should save pending changes when function throws an error', function() {
```

Comment:
The test name is confusing. It suggests that changes should be saved when the function throws an error -- but what function are you talking about? And the test doesn't seem to be checking/asserting that changes have been saved.

Reply:
Please make sure to respond to all comments before requesting a review. Thanks!

Reply:
I missed that, sorry!